processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -74,6 +77,76 @@
 {
   private volatile DateTime maxIngestedEventTime;
 
+  private static final Map<String, ValueType> TYPE_MAP = ImmutableMap.<String, ValueType>builder()
+      .put("Long[]", ValueType.LONG)
```

Comment:
@pjain1 It's not strictly necessary now, I think I had it in there from an older implementation where I used the TYPE_MAP to get a ValueType from a Comparable[]. 

I kept it there in case, but I can remove it if needed.